docs/designers-developers/developers/block-api/block-templates.md

```diff
@@ -17,18 +17,51 @@ Planned additions:
 
 Templates can be declared in JS or in PHP as an array of blockTypes (block name and optional attributes).
 
+The first example in PHP creates a template for posts that includes an image block to start, you can add as many or as few blocks to your template as needed.
+
+
+PHP example:
+
+```php
+<?php
+function myguten_register_template() {
+    $post_type_object = get_post_type_object( 'post' );
+    $post_type_object->template = array(
+        array( 'core/image' ),
+    );
+}
+add_action( 'init', 'myguten_register_template' );
+```
+
+The following example in JavaScript creates a new block which uses [InnerBlocks](/packages/editor/src/components/inner-blocks) and templates to insert a set of blocks:
+
 ```js
-const template = [
-  [ 'block/name', {} ], // [ blockName, attributes ]
+const el = wp.element.createElement;
+const { registerBlockType } = wp.blocks;
+const { InnerBlocks } = wp.editor;
+
+const BLOCKS_TEMPLATE = [
+	[ 'core/image', {} ],
+	[ 'core/paragraph', { placeholder: 'Image Details' }],
```

Comment:
For consistency, there should be space after closing `}` before `]`.

Reply:
👍